internal/webhooks/cluster.go

```diff
@@ -388,6 +388,9 @@ func (webhook *Cluster) validateTopology(ctx context.Context, oldCluster, newClu
 			allErrs = append(allErrs, check.ClusterClassesAreCompatible(oldClusterClass, clusterClass)...)
 		}
 	}
+
+	allErrs = append(allErrs, validateAutoscalerAnnotationsForCluster(newCluster, clusterClass)...)
```

Comment:
Let's move this directly after l.308. Just that we run it before the update-specific validation

(no strong ordering constraint just a preference to have it close to ValidateClusterForClusterClass)